container-images/hugo/README.adoc

```diff
@@ -4,6 +4,32 @@ This container image is provided to help test and build this hugo site.
 
 == Building the Image
 
+The hugo image is built in several stages:
+
+1. Build the rubygem-compiler
+2. Use the rubygem-compiler image to build an asciidoctor binary
+3. Build the hugo image
+
+Build the rubygem-compiler image:
+
+[source,bash]
+----
+docker build -t redhatcop/rubgem-compiler ./container-images/rubygem-compiler
```

Comment:
Spelling error in tag name (rubgem -> rubygem). Should be `redhatcop/rubygem-compiler`